src/shared/components/resource_sort_dropdown/generateSortItems.ts

```diff
@@ -401,15 +401,15 @@ export const generateSortItems = (
           sortDirection: Sort.Descending,
         },
         {
-          label: 'Description (Ascending)',
-          sortKey: 'description',
-          sortType: SortTypes.String,
+          label: 'Updated (Oldest)',
+          sortKey: 'updatedAt',
+          sortType: SortTypes.Date,
           sortDirection: Sort.Ascending,
         },
         {
-          label: 'Description (Descending)',
-          sortKey: 'description',
-          sortType: SortTypes.String,
+          label: 'Updated (Newest)',
+          sortKey: 'updatedAt',
+          sortType: SortTypes.Date,
```

Comment:
this looks like a shared component. Will changing this affect other pages?

Reply:
if you look on line 389 this is just the definition for subscriptions